### ISSUE TITLE
[ Software paper ] minor suggestions

### ISSUE THREAD

**Issue by hannahbaumann** [JustinYKC/FEPOPS#22]
Refers to the JOSS submission review: https://github.com/openjournals/joss-reviews/issues/5763:

Some small suggestions on the software paper:

- For the acronyms FEPOPS and AUROC it would be helpful to add a brief definition/spell out the acronyms upon first mentioning in the text
- line 84: typo: realatively --> relatively
- line 110: Add a brief explanation on how the macro-averaged AUROC scores were computed.
- results part (lines 108 and following): Add a sentence briefly comparing the performance of the different methods (Table 1)

**Comment by stevenshave** [comment]
Thank you for your comments and suggestions, we have now integrated all of them into the submission.

Typo addressed and acronyms are listed in full upon their first occurrence and we have added a citation breaking down the AUROC and ROC metrics.

We have removed the term macro from "macro averaging" throughout the paper, the README.md ,and online docs as it was not necessary here. We have clarified the process used in averaging AUROC scores in the paper text.

In addition, we have a short comparative summary of metric performance and conclusion.

Many thanks,
Steve
